hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/ITTestHoodieDataSource.java

```diff
@@ -147,20 +157,17 @@ void testStreamWriteAndReadFromSpecifiedCommit(HoodieTableType tableType) throws
         .option(FlinkOptions.READ_START_COMMIT, firstCommit)
         .end();
     streamTableEnv.executeSql(hoodieTableDDL);
-    List<Row> rows = execSelectSql(streamTableEnv, "select * from t1", 10);
-    assertRowsEquals(rows, TestData.DATA_SET_SOURCE_INSERT);
+    execSelectSqlAndCheckResult(streamTableEnv, "select * from t1", TestData.DATA_SET_SOURCE_INSERT);
```

Comment:
Maybe we use the current style and just extend the `execSelectSql` method.

Reply:
Do you mean keep the function name unchanged, and just do checking logic inside `execSelectSql`?